components/aws/sagemaker/common/_utils.py

```diff
@@ -528,6 +588,21 @@ def create_endpoint_config(client, args):
     except ClientError as e:
         raise Exception(e.response['Error']['Message'])
 
+def update_endpoint(client, region, endpoint_name, endpoint_config_name):
+  logging.info("Updating endpoint_name=" + endpoint_name)
+  try:
+      update_endpoint_response = client.update_endpoint(
+          EndpointName=endpoint_name,
+          EndpointConfigName=endpoint_config_name,
+          )
+      logging.info("Updated endpoint with name: " + endpoint_name)
```

Comment:
need more information here: this might be too soon to print this message ?

I think the endpoint will go into `Updating` status after `update_endpoint` is called and we need to wait for that phase to complete before printing this

Reply:
Changed the message to updating